### PR TITLE
Test/fixtures

### PR DESCRIPTION
**Pull request by iadmytro**
# Context

Updated test fixtures

### CODE REVIEW

File: packages/cardano-services/test/Asset/DbSyncAssetProvider.test.ts [hunk 1]
```diff
@@ -104,10 +104,10 @@ describe('DbSyncAssetProvider', () => {
 
     expect(asset.history).toEqual(history);
     // TODO: review test data, this is false positive right now
-    // expect(asset.nftMetadata).toBeTruthy();
+    expect(asset.nftMetadata).toBeTruthy();
     expect(asset.nftMetadata).toStrictEqual(assets[0].metadata);
     expect(asset.tokenMetadata).toStrictEqual({
-      assetId: '728847c7898b06f180de05c80b37d38bf77a9ea22bd1e222b8014d964e46542d66696c6573',
+      assetId: '17ebe33f8aeee1fe9a7277fe0dc02261531a896a8b89457895fe60294349502d303032352d7632',
```
Comment (iccicci) [inline packages/cardano-services/test/Asset/DbSyncAssetProvider.test.ts:110]: It is better to read this `assetId` from the DB in the program, otherwise next time we generate new fixtures this will fail again.

**Review by iccicci — CHANGES_REQUESTED** [review]
Good job! Just one more minor improvement request

File: packages/core/src/Provider/providerUtil.ts [hunk 2]
```diff
@@ -15,6 +15,7 @@ export const withProviderErrors = <T extends {}>(providerImplementation: T, toPr
   }, {} as any) as T;
 
 const tryParseBigIntKey = (key: string) => {
+  if (key.startsWith('0x')) return key.slice(2);
```
Comment (iccicci) [inline packages/core/src/Provider/providerUtil.ts:18]: Better to do a separate commit for this and adding an inline comment to explain why; if I remember correctly to support CIP-005 v2

**Review by iccicci — CHANGES_REQUESTED** [review]
Sorry, one more comment

**Review by iccicci — APPROVED** [review]
GJ!

File: packages/cardano-services/test/Asset/DbSyncAssetProvider.test.ts (hunk 1 above)
Comment (rhyslbw) [reply packages/cardano-services/test/Asset/DbSyncAssetProvider.test.ts:110]: Why is this resolved @iccicci ? Seems like legitimate feedback

Comment (rhyslbw) [reply packages/cardano-services/test/Asset/DbSyncAssetProvider.test.ts:110]: Oh, it's mocked :+1:

File: packages/cardano-services/test/ChainHistory/ChainHistoryHttpService.test.ts [hunk 3]
```diff
@@ -429,7 +429,7 @@ describe('ChainHistoryHttpService', () => {
 
       it('includes transactions only in specified block range', async () => {
         const { addresses, blockRange, txInRangeCount } = await fixtureBuilder.getAddressesWithSomeInBlockRange(2, {
-          lowerBound: Cardano.BlockNo(200),
+          lowerBound: Cardano.BlockNo(600),
```
Comment (rhyslbw) [inline packages/cardano-services/test/ChainHistory/ChainHistoryHttpService.test.ts:432]: This could be handled dynamically with a fixture, but let's not get bogged down here.

**Review by rhyslbw — APPROVED** [review]
Thanks @iadmytro
